parser2.py

```diff
@@ -0,0 +1,223 @@
+from datetime import datetime, timedelta
+import re
+import openpyxl
+from rich.console import Console
+from ics import Calendar, Event
+
+console = Console()
+
+
+def get_cell_value(cell):
+    if isinstance(cell.value, str):
+        return cell.value.strip()
+    elif isinstance(cell.value, (int, float)):
+        return str(cell.value)
+    elif isinstance(cell.value, bool):
+        return str(cell.value)
+    elif isinstance(cell.value, datetime):
+        return cell.value.strftime("%Y-%m-%d %H:%M:%S")
+    elif cell.value is None:
+        return ""
+    else:
+        return "Unknown Value"
```

Comment:
I mean. What is the point? Why dont you just do something like:
```suggestion
def get_cell_value(cell):
    return str(cell.value).strip if cell.value is not None else ""
```


Also, by any chance if you have some time consider reading about what is OOP (Overly Obsessive Programming)